src/main/java/com/thealgorithms/strings/Palindrome.java

```diff
@@ -11,27 +11,27 @@ class Palindrome {
     public static void main(String[] args) {
         String[] palindromes = { null, "", "aba", "123321" };
         for (String s : palindromes) {
-            assert isPalindrome(s) &&
+            assert isPalindromeStringBuilder(s) &&
             isPalindromeRecursion(s) &&
-            isPalindrome1(s);
+            isPalindromeTwoPointer(s);
         }
 
         String[] notPalindromes = { "abb", "abc", "abc123" };
         for (String s : notPalindromes) {
-            assert !isPalindrome(s) &&
+            assert !isPalindromeStringBuilder(s) &&
             !isPalindromeRecursion(s) &&
-            !isPalindrome1(s);
+            !isPalindromeTwoPointer(s);
         }
     }
 
     /**
-     * Check if a string is palindrome string or not
+     * Check if a string is palindrome string or not using String Builder
      *
      * @param s a string to check
      * @return {@code true} if given string is palindrome, otherwise
      * {@code false}
      */
-    public static boolean isPalindrome(String s) {
+    public static boolean isPalindromeStringBuilder(String s) {
```

Comment:
```suggestion
    public static boolean isPalindrome(String s) {
```
change the name to original one as it is the traditional method

Reply:
> change the name to original one as it is the traditional method

All Done. Added Test Cases, Removed main(), and changed the name of isPalindromeStringBuilder() to isPalindrome()